### DISCUSSION TITLE
What is the status of FutureSoot?

### DISCUSSION THREAD

**Discussion by paganma** [soot-oss/soot#1909] [Q&A] [answered]
I notice that there were plans for a legacy-free soot at http://soot-oss.github.io/soot/future-soot/, but I could not find any more information on this project. Is this project still in development?

**Comment by StevenArzt** [comment, marked as answer]
The project is ongoing, the core collaborators will hold a workshop on the matter next week. The topic isn't easy, because many projects depend on Soot and FutureSoot will likely not be backwards-compatible.

**Reply by paganma** [reply]
Thanks for the update. Will the workshop be online? If so, where can I learn more about it?

**Reply by StevenArzt** [reply]
I guess it's internal, but there might be some communication once we have made the relevant decisions.

**Reply by paganma** [reply]
I was pleased to notice that the FutureSoot page has been updated with a few more details. Is there any chance that the development repository will be released to the public anytime soon?

**Comment by ericbodden** [comment]
The project has been released as SootUp: https://soot-oss.github.io/SootUp/
